package.json

```diff
@@ -76,18 +78,21 @@
     "husky": "^4.2.5",
     "identity-obj-proxy": "^3.0.0",
     "jest": "^26.6.3",
+    "lodash": "^4.17.20",
```

Comment:
What do we need Lodash for?

Reply:
https://github.com/readmeio/markdown/pull/94/files#diff-58d8608f00398d6dfc613a4961f1641a01f9320dc5e72a19f5689d2c9a6e4cbeR27

Reply:
[Any reason to not use vanilla JS here?](https://github.com/readmeio/markdown/pull/94#discussion_r543694504)